app/routes/session.py

```diff
@@ -132,7 +132,16 @@ def get_sign_out():
     """
     Signs the user out of eQ and redirects to the log out url.
     """
-    log_out_url = get_survey_config().account_service_log_out_url
+    todo = request.args.get("todo", "").upper() == "TRUE"
```

Comment:
Maybe the default should be "FALSE" to keep the bool intention explicit rather than imply a tristate is possible (unless it is possible...)
```suggestion
    todo = request.args.get("todo", "FALSE").upper() == "TRUE"
```

Reply:
Also is it intentional that the string passed to the query param is case insensitive? It probably doesn't matter, just checking.

Reply:
We kept it this way because of possible discrepancies between python and jinja templates, I think, so it always evaluates to correct bool.